superset-frontend/src/components/AlteredSliceTag/index.jsx

```diff
@@ -32,6 +32,16 @@ const propTypes = {
   currentFormData: PropTypes.object.isRequired,
 };
 
+const StyledLabel = styled.span`
+  ${({ theme }) => `
+    font-size: ${theme.typography.sizes.s}px;
+    background-color: ${theme.colors.alert.base};
+
+    &: hover {
+      background-color: ${theme.colors.alert.dark1};
+    };`}
```

Comment:
```suggestion
  ${({ theme }) => `
    font-size: ${theme.typography.sizes.s}px;
    background-color: ${theme.colors.alert.base};

    &: hover {
      background-color: ${theme.colors.alert.dark1};
    }
  `}
```

Reply:
Is it really working with a side between &: and hover?